src/components/spaces/organisms/organelle-wrapper.sass

```diff
@@ -10,11 +10,11 @@
 
 .hover-location
     position: absolute
-    bottom: 5px
+    top: 45px
     right: 5px
     min-width: 94px
     height: 18px
-    background: #FFFFFF88
+    background: #FFFFFFAA
     border: 1px solid #444
```

Comment:
This reminds me that at some point we should remember to clean up all rogue colors and only use colors from the spec - but that can wait for another day.